app/controllers/kaui/custom_fields_controller.rb

```diff
@@ -39,8 +43,93 @@ def pagination
 
   def new
     @custom_field = Kaui::CustomField.new
+
   end
 
+  def check_object_exist
+
+    param_uuid = params[:uuid]
+
+    begin
+      test_uuid = Kaui::Invoice.find_by_invoice_item_id(param_uuid,false,'NONE', options_for_klient)
+
+    rescue StandardError
+    ensure
+      if !test_uuid.blank?
+        msg = { status: '200', message: I18n.translate('custom_field_uuid_exist_in_invoice_item_db')  }
```

Comment:
Few thoughts:

1. I don't think we should display anything if the object (e.g. invoice item) exists, just a warning if it doesn't
2. If the target object for the custom field doesn't exist, we should prevent the user from being able to save it
3. Can we send the object type to the back-end? That way, we don't have to try all endpoints (invoice item, bundle, etc.). That might take a while 😅

Reply:
Yes, good points. Got it.

Reply:
1. Done.
2. Pending.  
3. Done. Way better. 
There will be some improvements. 
https://github.com/killbill/killbill-admin-ui/pull/319/commits/11b43ff5f25250dcd38f94e9eb17d046f2ab56ef

Reply:
2. Done

Reply:
2. Was already done but response was from KillBill API without I18N and raw data.